options/rtdl/generic/main.cpp

```diff
@@ -118,9 +116,41 @@ extern "C" [[gnu::alias("dl_debug_state"), gnu::visibility("default")]] void _dl
 // This symbol can be used by GDB to find the global interface structure
 [[ gnu::visibility("default") ]] DebugInterface *_dl_debug_addr = &globalDebugInterface;
 
+namespace {
+bool log_entry_exit;
+bool log_startup;
+
+/// Returns weather the given environment variable (`env`) is present and
+/// non-empty.
+bool nonzero_env(char** rtdl_envp, frg::string_view env) {
+	if (!rtdl_envp) return false;
```

Comment:
Our coding style says that you shouldn't do control flow on the same line as an if statement.